README.md

```diff
@@ -61,15 +61,26 @@ install.packages('devtools')
 devtools::install_github('IOHprofiler/IOHanalyzer')
 ```
 
+### Development version
+
+If you want to run the version on which you develop:
+
+```r
+install.packages('devtools')
+devtools::install_git("/path/to/your/IOHanalyzer/git/repo")
+```
+
 ## <a name="run"></a>Runinng the Web Interface locally
 
-The IOHanalyzer package can be installed and loaded using the following commands:
+The IOHanalyzer package can be loaded using the following commands:
 
 ```r
 library('IOHanalyzer')
 runServer()
 ```
 
+It should open the following URL in your browser: [http://localhost:6582](http://localhost:6582).
```

Comment:
This is not entirely correct, the port number will be chosen by shiny, and will not always be 6582. To have the server open in a specific port, you need to explicitly provide this port-number to the runServer() command.